Makefile

```diff
@@ -63,19 +63,8 @@ upload-coverage:
 	go install github.com/mattn/goveralls@v0.0.11
 	/go/bin/goveralls -coverprofile=coverage.txt -service=drone.io
 
-
-
-BUILD_DT:=$(shell date +%F-%T)
-GO_LDFLAGS:="-s -w -extldflags \"-static\" -X main.BuildVersion=${DRONE_TAG} -X main.BuildCommitSha=${DRONE_COMMIT_SHA} -X main.BuildDate=$(BUILD_DT)" 
-OS ?= linux
-ARCH ?= amd64
-
 .PHONE: build-binaries
 build-binaries:
-	go install github.com/oliver006/gox@master
-
-	rm -rf .build | true
+	go install github.com/goreleaser/goreleaser/v2@latest
 
-	export CGO_ENABLED=0 ; \
-	gox -os=$(OS) -arch=$(ARCH) -verbose -rebuild -ldflags $(GO_LDFLAGS) -output ".build/kvrocks_exporter" && \
-	echo "done"
+	goreleaser release --clean -f .github/goreleaser.yml
```

Comment:
Could we separate the `build` and `release` binary into two targets? That said, we can have `make build` to build to binary which doesn't depend on goreleaser, and add the target `make release` for releasing propose. What do you think about this?

Reply:
Sure, made that change. My opinion I guess would be that this goreleaser step is never used and the release is only done by CI. Good then that the make build only makes a local build with no dependencies.

Reply:
And thanks for checking the PR - I'd suggest that assuming it works, I'd make another PR to update the readme a bit to also point to the docker hub reference.

Reply:
Sure, thank you!